forum/qa-include/qa-base.php

```diff
@@ -207,6 +202,12 @@ function qa_initialize_constants_2()
 
 	//	Default values if not set in qa-config.php
 
+        defined('QA_MYSQL_HOSTNAME') || define('QA_MYSQL_HOSTNAME', 'localhost');
+        defined('QA_MYSQL_USERNAME') || define('QA_MYSQL_USERNAME', 'test');
+        defined('QA_MYSQL_PASSWORD') || define('QA_MYSQL_PASSWORD', 'test');
+        defined('QA_MYSQL_DATABASE') || define('QA_MYSQL_DATABASE', 'forum');
+        defined('QA_MYSQL_TABLE_PREFIX') || define('QA_MYSQL_TABLE_PREFIX', 'qa_');
+        defined('QA_EXTERNAL_USERS') || define('QA_EXTERNAL_USERS', false);

```

Comment:
nie rozumiem czemu ta zmiana jest tutaj

Reply:
jak napisałem wyżej - zapoznaj się jak to rozwiązanie ma działać, nie tylko z samym kodem. Gdy pliku z konfiguracją nie ma to weźmie wartości domyślne, konfiguracją można sobie je nadpisać jak ktoś potrzebuje. Uproszczenie aby nie było wielu plików, bo ze stałymi i mechanizmem jaki tu mamy średnio to wygodne